### ISSUE TITLE
How to use this module for *.ts files?

### ISSUE THREAD

**Issue by pubkey** [smartprix/babel-changed#2]
Hi,
this module looks promising. Atm I use my custom script for cached transpilation, but I want to replace that with your module.

The problem is, I have a typescript project and want to make `babel-changed` to transpile my .ts files but they are ignored.

My current bash
```bash
babel-changed --src src --dest dist/lib --file-glob \"**/*.ts\"
```

Output:

```
> babel-changed --src src --dest dist/lib --file-glob "**/*.ts"

[babel] copying 76 files
[babel] nothing to compile
babel-changed: 56.8ms
```

From the output it looks like the ts-files are only copied, not transpiled.

EDIT:

Ok so it looks like this is intentional. https://github.com/smartprix/babel-changed/blob/master/index.js#L44
Babel has the [config option](https://babeljs.io/docs/en/babel-preset-typescript) `--extensions ".ts"` so maybe we should also add this option to babel-changed.

**Comment by rohit-gohri** [comment]
That sounds like a good idea. It'd be great if you could open a PR. Else I can take a look later.

CC: @hit12369 Sounds good?

**Comment by hit12369** [comment]
Released in `0.3.0`

**Comment by pubkey** [comment]
@rohit-gohri thank you for adding the extensions flag. The files are running through the transpilation now.
Sadly I now have another problem. The output files are not `.js` but still `.ts` files.
With plain babbel, I think babbel assumes that everything that is transpiled, results in a js file.

**Comment by rohit-gohri** [comment]
Opened another PR #5 @hit12369 
Tested it this time on a sample project 😅

**Comment by hit12369** [comment]
Released in `0.4.0`
